reports/report_builder.py

```diff
@@ -1240,7 +1240,7 @@ def ws_28b(self, ws):
         c = 1
         r = 8
         write_row_headings = True
-        all_regions = add_transnational_to_regions(self.regions)
+        all_regions = add_transnational_to_regions(self.regions) if self.report_type == 'global' else self.regions
```

Comment:
This is needed in [ws_28](https://github.com/CodeForAfrica/gmmp/pull/175/files#diff-19a686dbe88bb9f2f8c4f2ce5e331aa6f6db260926a2a5b7f1edfa7864a649b4L1186) as well.